rtp/RtpDelayTracker.cpp

```diff
@@ -0,0 +1,58 @@
+#include "rtp/RtpDelayTracker.h"
+#include "utils/Time.h"
+#include <algorithm>
+
+namespace rtp
+{
+
+RtpDelayTracker::RtpDelayTracker(uint32_t rtpFrequency, uint32_t clockSkewRtpTicks)
+    : _frequency(rtpFrequency),
+      _clockSkewCompensation(clockSkewRtpTicks),
+      _renderTime(0),
+      _rtpTimestamp(0)
+{
+}
+
+uint64_t RtpDelayTracker::update(uint64_t receiveTime, uint32_t rtpTimestamp)
+{
+    if (_renderTime == 0 && _rtpTimestamp == 0)
+    {
+        _renderTime = receiveTime;
+        _rtpTimestamp = rtpTimestamp;
+        _delay = 0;
+        return 0;
+    }
+
+    _renderTime += static_cast<int32_t>(rtpTimestamp - _rtpTimestamp) * static_cast<int64_t>(utils::Time::sec) /
```

Comment:
This is calculating the renderTime with slight lower frequency. So if the real frequency is correct or higher, that will increasing the reported delay over time, right? I don't find a point where this can be fixed for more real values in that case. How that affects the final jitter measurement?